public/js/index.js

```diff
@@ -1675,6 +1675,26 @@ function initVueComponents(){
                 type: Number,
                 required: true
             },
+            orgs: {
```

Comment:
All variables are "filled by server" in `dashboard.tmpl` when creating Vue component and then used inside `repo-search.tmpl`.